### DISCUSSION TITLE
how to automatically open the index.mdx when click sidebar menu?

### DISCUSSION THREAD

**Discussion by kobe651jp** [shuding/nextra#1750] [Q&A] [answered]
Clicking on the sidebar menu of  [nextra's documentation site](https://nextra.site/docs) automatically open the index.mdx.
But on my site the same operation just opens and closes the menu.

what am i missing? Is there a way to handle this behavior?

https://user-images.githubusercontent.com/44263972/231326769-d20ab48a-0f29-41ac-a2c5-3be8e6abd532.mp4

**Comment by cglacet** [comment, marked as answer]
I think its not in the doc, but if you look at the documentation's source itself, you'll notice that the trick is to have (in `pages/docs`) both a [`guide/` folder](https://github.com/shuding/nextra/tree/main/docs/pages/docs/guide) and a [`guide.mdx` file](https://github.com/shuding/nextra/blob/main/docs/pages/docs/guide.mdx).

In the `pages/docs/_meta.json` they have simply this:

```json
{
  "index": "Introduction",
  "guide": "",
  
}
```

**Reply by kobe651jp** [reply]
You are right!
Using that trick I was able to confirm that my site is also working fine.
